utils/chain-ops/genesis/layer_one.go

```diff
@@ -272,7 +254,8 @@ func BuildL1DeveloperGenesis(config *DeployConfig) (*core.Genesis, error) {
 	if err != nil {
 		return nil, err
 	}
-	data, err = securityCouncilTokenABI.Pack("initialize", config.SecurityCouncilTokenOwner)
+
+	data, err = securityCouncilTokenABI.Pack("initialize", opts.From)
```

Comment:
Why do you change this token owner to deployer?

Reply:
I used it because it is an EOA that can do minting in this block.